testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -382,8 +385,8 @@ public void verifyCompressionCodecsDataProvider()
         assertThat(onTrino().executeQuery("SHOW SESSION LIKE 'delta.compression_codec'"))
                 .containsOnly(row(
                         "delta.compression_codec",
-                        "SNAPPY",
-                        "SNAPPY",
+                        trinoParquetCompressionCodec,
+                        trinoParquetCompressionCodec,
```

Comment:
from `verifyCompressionCodecsDataProvider` perspective, this doesn't matter
so you can use `anyOf("SNAPPY", "ZSTD")` here 

what matters is the fact that `compressionCodecs()` matches the list of available codecs from the connector.